benchmarks/dynamo/common.py

```diff
@@ -1388,6 +1400,10 @@ def __init__(self, output_directory, model, example_inputs, dynamic_shapes: bool
         self.onnx_session = self._init_ort_session(self.model_path)
 
     def _export(self, model, example_inputs, output_path: str, /, **kwargs) -> None:
+        if self.copy_before_export:
+            # Deepcopy model before export to avoid modification to baseline model.
+            model = copy.deepcopy(model)
```

Comment:
So we have enough memory for this?

Reply:
We are doing more deepcopies for accuracy bench already. Perf bench will require less memory even with this change. So it shouldn't impose any regression.
That being said, deepcopies is indeed the one of the contributing factors for OOM in benches. We need it to ensure accurate perf/accuracy metric. I'm open to ideas how to improve this aspect.